src/sentry/mediators/external_requests/util.py

```diff
@@ -35,3 +40,26 @@ def validate(instance, schema_type):
         return False
 
     return True
+
+
+def send_and_save_request(sentry_app, org_id, event, url, **kwargs):
```

Comment:
@nolachen I think something in the name of this function should reflect that it's only used for sentry apps.

Reply:
@scefali I thought that everything in /mediators is for Sentry Apps? Or is that not true

Reply:
@nolachen Nope, we have mediators for plugins and other things as well.

Reply:
@scefali just to make sure is `select_requester` and `issue_link_requester` just for Sentry Apps though?

Reply:
@nolachen That's true. But even if the mediators were only used for SentyrApps, there's nothing in the path that says that so someone who doesn't know that part of the code wouldn't know that.